frontend/src/components/pipelines-or-deployments/pipeline-service.js

```diff
@@ -0,0 +1,60 @@
+import { useAxiosPrivate } from "../../hooks/useAxiosPrivate";
+import { useSessionStore } from "../../store/session-store";
+
+function pipelineService() {
+  const axiosPrivate = useAxiosPrivate();
+  const { sessionDetails } = useSessionStore();
+  const path = `/api/v1/unstract/${sessionDetails.orgId.replaceAll('"', "")}`;
```

Comment:
Let's have optional chaining for sessionDetails too. in order for it to be safe to have and consistent throughout the code.

Reply:
@jagadeeswaran-zipstack Thanks for the catch. Done.